test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -1529,5 +1529,51 @@ public class ModelBindingTest
             var body = await response.Content.ReadAsStringAsync();
             Assert.Equal(expectedContent, body);
         }
+
+        [Fact]
+        public async Task TryUpdateModelIncludesAllProperties_CanBind()
```

Comment:
use consistent names, should this really be
```TryUpdateModelNonGeneric_CanBind````, however it would be more interesting to see a type which is different from the actual runtime type. Add one if there is none already.